client/state/action-types.ts

```diff
@@ -373,6 +373,12 @@ export const JETPACK_CONNECTION_STATUS_REQUEST_FAILURE =
 	'JETPACK_CONNECTION_STATUS_REQUEST_FAILURE';
 export const JETPACK_CONNECTION_STATUS_REQUEST_SUCCESS =
 	'JETPACK_CONNECTION_STATUS_REQUEST_SUCCESS';
+export const JETPACK_CONNECTION_HEALTH_REQUEST = 'JETPACK_CONNECTION_HEALTH_REQUEST';
+export const JETPACK_CONNECTION_HEALTH_REQUEST_RECEIVE = 'JETPACK_CONNECTION_HEALTH_RECEIVE';
```

Comment:
This action type seems unused.

Also it's a potentially difficult to track issue when the action type name isn't equal to the action type value.